src/dream.mli

```diff
@@ -1545,8 +1544,8 @@ val no_route : route
 (** {1 Static files} *)
 
 val static :
-  ?loader:(string -> string -> handler) ->
-    string -> handler
+  ?loader:('a Eio.Path.t -> string -> handler) ->
+    'a Eio.Path.t -> handler
```

Comment:
What's the advantage of using `'a Eio.Path.t` over `string` here? One of the goals of Dream is to avoid arcane types and to be as immediately comprehensible as reasonably possible, and having this type here is opposed to that.

Reply:
Using `Eio.Path.t` was the minimal way to get a capability into the function. We need a capability to read from a fs and without any access to env we need the root to be a path.

Regarding removing the loader argument: What possible use cases would be affected by that? I see for example that the standard loader ignores the request. If it could no longer be replaced, more advanced handlers have to wrap the `static` function instead. I assume this would not be a problem?